migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,9 @@
+var SafeMath = artifacts.require("./SafeMath.sol");
+var Splitter = artifacts.require("./Splitter.sol");
+
+var bob = "0x978ddD545eA2a1C73B690Fa8557Edf748e385252";
+var carol = "0x2c6A4E13E56c93B86aE9bf6030E6C1062f61ff62";
```

Comment:
Don't hard-code. Remember that `module.exports = function(deployer, network, accounts) {`.